cmd/webhook/app/webhook.go

```diff
@@ -103,6 +105,9 @@ func Run(ctx context.Context, opts *options.Options) error {
 		LeaderElection:         false,
 		MetricsBindAddress:     opts.MetricsBindAddress,
 		HealthProbeBindAddress: opts.HealthProbeBindAddress,
+		NewCache: cache.BuilderWithOptions(cache.Options{
+			DefaultTransform: util.StripUnusedFields,
+		}),
```

Comment:
I think no informer actually running in webhook. So, why do we set this for it?

Reply:
Thanks for reminding, removed